\noindent
\large {\bf MICADo - Looking for mutations in PacBio cancer data: an alignment-free method} 


\normalsize 


\noindent \paragraph{Keywords:} PacBio, Cancer Genomics, Next-generation sequencing, Mutation
Calling, de Bruijn graphs

\noindent \paragraph{Abstract:} 
Targeted sequencing is commonly used in clinical application of NGS technology since it enables generation
of sufficient sequencing depth in the targeted genes of interest and thus ensures the best possible downstream analysis.
This notwithstanding, the accurate discovery and annotation of disease causing mutations remains a challenging problem
even in such favorable context. The difficulty is particularly salient in the case of third generation sequencing technology,
such as PacBio.

We present MICADo, a de Bruijn graph based method that makes possible to distinguish between patient specific
mutations and other alterations for targeted sequencing of a cohort of patients. MICADo analyses NGS reads for each
sample within the context of the data of the whole cohort in order to capture the differences between specificities of the
sample with respect to the cohort. MICADo is particularly suitable for sequencing data from highly heterogeneous samples,
especially when it involves high rates of non-uniform sequencing errors. It was validated on PacBio sequencing datasets
from several cohorts of patients.

Availability: The source code is available at \url{http://github.com/cbib/MICADo}.


\noindent \paragraph{Authors:} 

\noindent \paragraph{} 

{
\centering
\resizebox{\textwidth}{!}{%
\begin{tabular}{|l|l|l|l|l|l|}
\hline
\textbf{first name} & \textbf{last name} & \textbf{email}            & \textbf{country} & \textbf{organization}                                                                   & \textbf{corresponding?} \\ \hline
Justine             & Rudewicz           & justinerudewicz@gmail.com & France           & \begin{tabular}[c]{@{}l@{}}University of Bordeaux,\\ CNRS/LaBRI and INSERM\end{tabular} &  $\checkmark$                       \\ \hline
Hayssam             & Soueidan           &                           & France           & \begin{tabular}[c]{@{}l@{}}University of Bordeaux \\ and CNRS/LaBRI\end{tabular}        &                         \\ \hline
Raluca              & Uricaru            &                           & France           & \begin{tabular}[c]{@{}l@{}}University of Bordeaux \\ and CNRS/LaBRI\end{tabular}        &                         \\ \hline
Herv√©               & Bonnefoi           &                           & France           & INSERM                                                                                  &                         \\ \hline
Richard             & Iggo               &                           & France           & INSERM                                                                                  &                         \\ \hline
Jonas               & Bergh              &                           & Sweden           & Karolinska Institute                                                                    &                         \\ \hline
Macha               & Nikolski           & macha.nikolski@labri.fr   & France           & \begin{tabular}[c]{@{}l@{}}University of Bordeaux\\ and CNRS/LaBRI\end{tabular}         &  $\checkmark$                       \\ \hline
\end{tabular}%
}
}

