\noindent
\large {\bf scphaser: haplotype inference using single cell RNA-seq data} 


\normalsize 


\noindent \paragraph{Keywords:} 

\noindent \paragraph{Abstract:} 

Determination of haplotypes is important for correctly modelling the phenotypic consequences of genetic
variation in diploid organisms, including cis-regulatory control and compound heterozygosity. Single cell
RNA-seq (scRNA-seq) data is exceptionally suited for phasing genetic variants, since both
transcriptional bursts and technical bottlenecks cause pronounced allelic fluctuations in individual single
cells. Here we present scphaser, an R package that phases alleles at heterozygous variants to
reconstruct haplotypes within transcribed regions of the genome using scRNA-seq data. The devised
method efficiently and accurately reconstructed the known haplotype for $\geq$93\% of phasable genes in
both human and mouse. It also enables phasing of rare and de novo variants and variants far apart
within genes, which is hard to attain with population-based computational inference. scphaser is
implemented as an R package. Tutorial and code are available
\url{https://github.com/edsgard/scphaser} (Private repository, access available upon request).

\noindent \paragraph{Authors:} 

